jbmc/src/java_bytecode/java_class_loader.cpp

```diff
@@ -102,6 +107,10 @@ optionalt<java_bytecode_parse_treet> java_class_loadert::get_class_from_jar(
   return java_bytecode_parse(istream, get_message_handler());
 }
 
+/// Check if class is an overlay class by inspecting the `ID_overlay_class`
```

Comment:
Add a link to something that defines what an overlay class is - @NathanJPhillips might know a suitable thing to link to?

I'd also not include how the function works as that is just liable to go out of date.

Reply:
talked to @NathanJPhillips , will add ad todo for him for some later documentation effort and will also extend the comment to how this works